.travis.yml

```diff
@@ -58,10 +58,6 @@ script:
   # Javascript and python linting
   - gulp lint
 
-  # make migrations
```

Comment:
Could you please add clarity on why this PR is making changes to `.travis.yml`

Reply:
@collin5  The reason the PR is making changes to the .travis.yml is because travis would stop to      
run builds whenever it reached this particular line. This was probably because the migration scripts  that i pushed were in conflict with those used by travisCI . As a matter of fact this line was added there in the first place to just force travisCI to run migrations because of the need to merge migrations on travisCI. But once the migrations were run and merged there was no point of rerunning migrations again.

Reply:
Hmm, ok. I think we could then leave out the `makemigrations`. Though we shouldn't get rid of the `python manage.py migrate` as this may under certain circumstanced be required to be run by travis before running tests.

Reply:
Ok thanks. Let me put back the python manage.py migrate line. 👍